setup.cfg

```diff
@@ -24,7 +24,7 @@ offline = True
 
 [flake8]
 # E501: line too long
-ignore = E501
+ignore = E501,W503
```

Comment:
For future clarity, please also add a comment after L26 to explain what W503 is.